seed/gallerySeeder.js

```diff
@@ -0,0 +1,51 @@
+var Gallery = require('../models/gallery');
+var mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost:27017/galleries');
+
+var gallery = [
```

Comment:
Make sure you are using all 100 data record instead of a sample from it.

Reply:
Will do. Right now just testing it.